### ISSUE TITLE
Can't apply DateFormat

### ISSUE THREAD

**Issue by Turbo87** [sir-dunxalot/ember-google-charts#11]
currently my code looks roughly like this:

``` js
var dataTable = window.google.visualization.arrayToDataTable(data);

var fmt = new window.google.visualization.DateFormat({
    pattern: 'LLL d, yyyy',
});
fmt.format(dataTable, 0);

chart.draw(dataTable, options);
```

I would like to use this library, but it seems that the components expect the raw data instead of the dataTable and I have no chance of applying the `DateFormat` class after it gets converted by the lib.

**Comment by sir-dunxalot** [comment]
So the fix would be to allow one to pass in a data table as well as or instead of raw data?

**Comment by Turbo87** [comment]
either that or provide some sort of `formatData` action hook that gets passed the `dataTable` as a parameter

**Comment by sir-dunxalot** [comment]
Ok, sounds good. I'll aim to add this as a feature.

**Comment by sir-dunxalot** [comment]
Fixed by #50.
